templates/default/logback.xml.erb

```diff
@@ -50,7 +50,6 @@
     </encoder>
   </appender>
 
-  <% if node['cassandra']['logback']['debug']['enable'] -%>
```

Comment:
Is this essential to this PR? I don't think we should enable debug logging by default?